packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -58,28 +60,50 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 	const mode = useSelect( ( select ) => {
 		return select( blockEditorStore ).getBlockMode( clientId );
 	} );
+	const { isPartOfSelection, adjustScrolling, enableAnimation } = useSelect(
```

Comment:
Shouldn't these `useSelect` calls be merged?

Reply:
Yes, already did it in c1209dd99592aea318ff135a2c50f3d5b061dd77, but didn't push it as I wanted to see the test results.